mypy/fastparse.py

```diff
@@ -21,7 +21,9 @@
 from mypy.errors import Errors
 
 try:
-    import typed_ast  # type: ignore
+    from typed_ast import ast27  # type: ignore
+    from typed_ast import ast35  # type: ignore
+    from typed_ast import conversions  # type: ignore
```

Comment:
Maybe you could provide stubs for these? The untypedness here kind of concerns me.